server/schedulers/balance_region.go

```diff
@@ -34,11 +34,11 @@ func init() {
 		return func(v interface{}) error {
 			conf, ok := v.(*balanceRegionSchedulerConfig)
 			if !ok {
-				return ErrScheduleConfigNotExist
+				return errs.ErrScheduleConfigNotExist
 			}
 			ranges, err := getKeyRanges(args)
 			if err != nil {
-				return errors.WithStack(err)
+				return errs.ErrSchedulerConfig.Wrap(err).GenWithStackByCause("ranges")
```

Comment:
Does this error need to be handled in the lower level?

Reply:
I think it is acceptable as a scheduler error at the moment cc@nolouch

Reply:
I think you should wrap it in `getKeyRanges` function.